prov/rxm/src/rxm_init.c

```diff
@@ -299,6 +299,13 @@ RXM_INI
 			"Messages of size greater than this (default: 256 Kb) "
 			"would be transmitted via rendezvous protocol.");
 
+	fi_param_define(&rxm_prov, "use_srx", FI_PARAM_BOOL,
+			"Set this enivronment variable to control the RxM "
+			"receive path. If this variable set to 1 (default: 0), "
+			"the RxM uses Shared Receive Context. This mode improves "
+			"memory consumption, but it can lead to poor performance "
```

Comment:
nit: say 'may increase small message latency' rather than 'can lead to poor performance'  :)

Reply:
Sure! Done